lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -136,6 +136,13 @@ class ConfigSchemaHandler {
             this.serverless.cli.log(`  ${message}`, 'Serverless', { color: 'orange' });
           }
         }
+        if (!this.serverless.configurationInput.configValidationMode) {
+          this.serverless._logDeprecation(
+            'CONFIG_VALIDATION_MODE_DEFAULT',
+            'Starting with next major, Serverless will throw on configuration errors by default. ' +
+              'Adapt to this behavior now by adding "configValidationMode: error" to service configuration'
+          );
+        }
```

Comment:
Log as propsed will be output in a middle of configuration warning (before actual listed warnings and `Learn about configuration validation...`  part)

Is this on purpose?

I think it might be nicer to have deprecation logged either right before the any scheme warnings or right after whole configuration validation section.

Reply:
@medikoo I thought it made sense because it was related to the ```Configuration warning at root``` error. I moved it to the bottom though, let me know what you think.

Reply:
Thanks for an update @yumeixox and thanks @medikoo for a good point here. I think it makes more sense in the current form, what are your thoughts @medikoo ?